Code/GraphMol/RascalMCES/Wrap/testRascalMCES.py

```diff
@@ -218,6 +218,22 @@ def testMinCliqueSize(self):
     mol2 = Chem.MolFromSmiles('CC12CCC3C(C1CCC2O)CCC4=C3C=CC(=C4)O')
     results = rdRascalMCES.FindMCES(mol1, mol2, opts)
     self.assertFalse(results)
+
```

Comment:
My personal preference is to have these functionality tests in the C++ (mces_catch.cpp in this case), and just have minimal testing in the Python layer to make sure things like options have been exposed correctly.  That's really one for @greglandrum to adjudicate, though, I think.

Reply:
I agree with @DavidACosgrove on this one.